gulpfile.js

```diff
@@ -112,6 +112,7 @@ declareExtension('amp-lightbox', '0.1', true);
 declareExtension('amp-lightbox-viewer', '0.1', true);
 declareExtension('amp-list', '0.1', false);
 declareExtension('amp-live-list', '0.1', true);
+declareExtension('amp-mathml', '0.1', false);
```

Comment:
Now we need a `CSS` file as well and `false` becomes `true`
The CSS file would handle the `inline` attribute by simply doing 
```
amp-mathml[inline] {
  display: inline-block;
}
```

Note: we _may_ need to do 
```
amp-mathml {
  height: 1px;
}
amp-mathml[inline] {
  width: 1px;
}
```
to make it not be 0by0 until resize happens. Try it without the above 1px hacks, if it works all is good. Otherwise if you see mathML not rendering, we may need to add them. (This is because AMP tends to ignore elements that are 0x0)

Reply:
The 1px settings were required, without that the iframe never rendered.